content/en/docs/reference/command-line-tools-reference/feature-gates/remove-self-link.md

```diff
@@ -12,11 +13,12 @@ stages:
     toVersion: "1.19"
   - stage: beta
     defaultValue: true
-    fromVersion: "1.20"  
-    toVersion: "1.23" 
+    fromVersion: "1.20"
+    toVersion: "1.23"
   - stage: stable
     defaultValue: true
-    fromVersion: "1.24"  
+    fromVersion: "1.24"
+    toVersion: "1.29"
```

Comment:
I don't much like the trailing whitespace, but: I'd rather leave it as-is. Makes merges easier.